docker-compose.yaml

```
@@ -29,16 +29,25 @@ services:
     env_file:
       - .env
 
-  nginx:
-    image: nginx:1.23.2-alpine
-    container_name: nginx
-    restart: always
+  swag:
+    image: lscr.io/linuxserver/swag
+    container_name: swag
+    restart: unless-stopped
+    cap_add:
+      - NET_ADMIN
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/Moscow
+      - URL=lombaryery.tk
+      - VALIDATION=http
     ports:
+      - "443:443"
       - "80:80"
     depends_on:
       - backend
     volumes:
       - static_value:/var/html/static/
-      - ./src/nginx/default.conf:/etc/nginx/conf.d/default.conf
+      - ./src/nginx/default.conf:/config/nginx/site-confs/default.conf
```

```suggestion
      - ./nginx/default.conf:/config/nginx/site-confs/default.conf
```

Так как в папке с проектом лежат докер файлы, .env файлы, data файлы, и папка src, в которой nginx.